integration_tests/plugins/index.ts

```diff
@@ -34,7 +34,23 @@ export default (on: (string, Record) => void): void => {
     },
 
     getSignInUrl: auth.getSignInUrl,
-    stubSignIn: auth.stubSignIn,
+
+    stubSignIn: () =>
```

Comment:
Maybe the name should be something a bit more generic - it doesn't just sign in. Maybe `stubSetup`.. actually I can't think of a good one yet...